docker/images/docker-spark/master/supervisord.conf

```diff
@@ -0,0 +1,39 @@
+[supervisord]
+logfile = /tmp/supervisord.log
+logfile_maxbytes = 50MB
+logfile_backups=10
```

Comment:
these means 10 copies?

Reply:
Yeah these were defaults that I used, I was actually considering getting rid of the entire section because these are the logs for supervisord itself not the Ingestion

Reply:
I ended up getting rid of all these settings. They seem completely unnecessary now.